Makefile

```diff
@@ -78,6 +78,9 @@ manifests: controller-gen
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+apidoc: apidocs-gen
+	$(APIDOCS_GEN) crdoc --resources config/crd/bases --output docs/content/general/references.md
```

Comment:
Rather than pointing to `docs/content/general/references.md`, couldn't we use a template to reference the resulting file in the `references.md` file that is useful since providing further information about Capsule?

e.g.:

```go
# Reference

Reference document for Capsule Operator configuration

## Custom Resource Definition

{{ .tenantCRDMarkdown }}

## Capsule Configuration

The Capsule configuration can be piloted by a Custom Resource definition named `CapsuleConfiguration`.

{{ .capsuleConfigurationCRDMarkdown }}
````